src/backend/TrafficCourts/Common/Features/Mail/Model/MailTemplateCollection.cs

```diff
@@ -66,6 +66,31 @@ public class MailTemplateCollection
                 HtmlContentTemplate = "TBD",
                 PlainContentTemplate = "TBD"
             },
+        new MailTemplate()
+            {
+                TemplateName = "DisputantUpdateRequestReceivedTemplate",
+                Sender = "DoNotReply@tickets.gov.bc.ca",
+                SubjectTemplate = "TBD (ie. Disputant contact information changes received)",
+                HtmlContentTemplate = "TBD",
+                PlainContentTemplate = "TBD"
+            },
+        new MailTemplate()
+            {
+                TemplateName = "DisputantEmailUpdateSuccessfulTemplate",
+                Sender = "DoNotReply@tickets.gov.bc.ca",
+                SubjectTemplate = "TBD (ie. Your email address is updated successfully)",
+                HtmlContentTemplate = "TBD",
```

Comment:
Subject "Email verification successful."  PlainContentTemplate "Your email address has been successfully updated."

Reply:
Done.

Reply:
I created TCVP-2017 for Colm to fill out for these details